### DISCUSSION TITLE
List of developers

### DISCUSSION THREAD

**Discussion by frojasg1** [community/community#31115] [Other Feature Feedback, Questions, & Ideas]
Hello,

    My question has to do with the number of developers that currently rely on github.
    I do not know if the following functionality already exists:

    I would like to access the list of developers, may be that could be queried by key words, as instance: "chess development", or just "chess"
    And that could be sorted by any kind of relevance, or filtered by user criteria...

    I would find very interesting to be able to visit their personal web pages or even to be able contact them.

    And that was my question.

    Thank you for offering your great service!

Regards,
Fran.

**Comment by jfullstackdev** [comment]
you can search them, like this 

https://github.com/search?q=chess+development

you will have all sorts of filters there, once you are at the page, you can change the keywords or specify the languages

![image](https://user-images.githubusercontent.com/47092464/187079329-2695867e-a585-4560-aa17-28d5003e1702.png)

**Reply by frojasg1** [reply]
Thank you, jdevstatic, for your fast answer.

Nice tool for filtering.
But I can see that normally developers do not fill their personal data, what a pity ...

**Reply by frojasg1** [reply]
How can I visit your web site??

;-)

**Comment by airtower-luna** [comment]
> I would find very interesting to be able to visit their personal web pages or even to be able contact them.

Some people provide web sites or mail addresses in their profile. However, mind that contacting people out of the blue (without good reason like privately reporting a security problem) is often considered rude.
